bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx110-en-features-v1.html

```diff
@@ -0,0 +1,126 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('firefox_whatsnew_110_en_features') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew110-features' %}
+{% set _utm_campaign = 'whatsnew110' %}
```

Comment:
Suggestion (non-blocking): should campaign be specific to the variation in some way? e.g. `whatsnew110-en-v1`?